haystack/nodes/other/join.py

```diff
@@ -21,7 +21,10 @@ def run(  # type: ignore
         top_k_join: Optional[int] = None,
     ) -> Tuple[Dict, str]:
         if inputs:
-            return self.run_accumulated(inputs, top_k_join=top_k_join)
+            results = self.run_accumulated(inputs, top_k_join=top_k_join)
+            if "root_node" in inputs[0]:
```

Comment:
Hey, @ZanSara I went ahead and added this if statement to stop the unit tests from failing.

Reply:
Thank you! I though I run the tests locally, but apparently not all of them :sweat_smile: